app.rb

```diff
@@ -1,7 +1,34 @@
 require 'sinatra/base'
+require 'sinatra/reloader' 
```

Comment:
Really nice readability in your code, I never felt lost going through it and can easily follow/understand whats happening!